### PR TITLE
fix: correctly handle IfStatement's alternate branch

### LINKED ISSUE #85: 🐛 Bug: If and else branch is printed incorrectly
### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/prettier-plugin-curly/issues?q=is%3Aissue) and found none that matched my issue.

### Expected

```ts
if (2) 1
else 2
```

### Actual

Is being printed as ...

```ts
if (2) { 1;
else 1; }
```

### Additional Info

_No response_

### PR DESCRIPTION
**Pull request by so1ve**
## PR Checklist

- [x] Addresses an existing open issue: fixes #85
- [x] That issue was marked as [`status: accepting prs`](https://github.com/JoshuaKGoldberg/prettier-plugin-curly/issues?q=is%3Aopen+is%3Aissue+label%3A%22status%3A+accepting+prs%22)
- [x] Steps in [CONTRIBUTING.md](https://github.com/JoshuaKGoldberg/prettier-plugin-curly/blob/main/.github/CONTRIBUTING.md) were taken

### CODE REVIEW

File: src/printNodeWithBrackets.ts [hunk 1]
```diff
@@ -53,9 +53,22 @@ export function printNodeWithBrackets(code: string, node: CollectibleNode) {
 			return [
 				code.slice(node.start!, node.test.end!),
 				") { ",
-				code.slice(node.consequent.start!, node.end!),
+				code.slice(node.consequent.start!, node.consequent.end!),
 				" }",
-			].join("");
+				node.alternate && [
+					" else { ",
```
Comment (auvred) [inline src/printNodeWithBrackets.ts:59]: I guess 

```js
if (a) b; else if (b) c;
```

will fail

`alternate` node with `IfStatement` requires additional handling https://github.com/prettier/prettier/blob/50404103ef7d96fb36f8cba1ac7bc40c671fd6cf/src/language-js/print/estree.js#L407

Comment (so1ve) [reply src/printNodeWithBrackets.ts:59]: Thanks! Updated.

**Review by JoshuaKGoldberg — APPROVED** [review]
I'm honestly surprised this hasn't come up yet 😄 thanks!

**Comment by JoshuaKGoldberg** [comment]
Oh nooooo it's supposed to be set by default on all my repos! #83

I'll rewrite Git history, one moment... 😄

**Pull request merged**